abb_irb120_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,31 @@
+joint_limits:
+  joint_1:
+    has_velocity_limits: true
+    max_velocity: 4.36332
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
Missing acceleration limits. I've seen strange planning results from MoveIt without them, but I don't know what the current state of this is.

I've always just either set them to the real limits (if available), or used the 'old way' of setting them to `1/joint_vel_limit`.